src/main/java/edu/ie3/datamodel/io/source/csv/CsvDataSource.java

```diff
@@ -90,6 +81,11 @@ public Stream<Map<String, String>> getSourceData(Class<? extends Entity> entityC
 
   // -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
 
+  /** Returns the set {@link FileNamingStrategy}. */
+  public FileNamingStrategy getNamingStrategy() {
+    return fileNamingStrategy;
+  }
+
```

Comment:
Is this on purpose?

Reply:
Yes. Befor we had a public method between protected methods and I thought it is better to have all public methods next to each other.

Reply:
Ah, I overlooked this. Good.